content/docs/user-guide/experiment-management/managing-experiments.md

```diff
@@ -0,0 +1,68 @@
+# Managing Experiments
+
+Once you have defined and/or [run experiments] in your project, you can use
+several features of DVC to see, compare, reproduce, share, or remove them.
+
+[run experiments]: /doc/user-guide/experiment-management/running-experiments
+
+## Experiment names
+
+Experiments created with `dvc exp run` will have an auto-generated name like
+`exp-bfe64` by default. It can be customized using the `--name` (`-n`) option:
+
+```dvc
+$ dvc exp run --name cnn-512 --set-param model.conv_units=512
+```
+
+When you create an experiment, DVC generates a Git-like SHA-1 hash from its
+contents. This is shown when you [queue experiments] with `--queue`:
```

Comment:
This is actually different from the hash generated after the experiments is committed, so I'm not sure it makes sense to mention it here.

Reply:
Moved this to https://github.com/iterative/dvc.org/pull/2752#pullrequestreview-737794776.